src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -35,12 +35,14 @@ class ObjectField extends React.Component {
       properties =>
         orderProperties(
           properties,
-          this.props.uiSchema['ui:order']?.filter(prop =>
-            // `view:*` properties will have been removed from the schema and
-            // values by the time they reach this component. This removes them
-            // from the ui:order so we don't trigger an error in the
-            // react-json-schema library for having "extraneous properties."
-            Object.keys(this.props.schema.properties).includes(prop),
+          this.props.uiSchema['ui:order']?.filter(
+            prop =>
+              // `view:*` properties will have been removed from the schema and
+              // values by the time they reach this component. This removes them
+              // from the ui:order so we don't trigger an error in the
+              // react-json-schema library for having "extraneous properties."
+              prop === '*' ||
```

Comment:
When `['serviceBranch', '*']` is used, what is the intention of the `*`? Does the entire thing mean "put `serviceBranch` first, and anything else after that"?

Reply:
Yes, that's what the documentation says - https://github.com/department-of-veterans-affairs/react-jsonschema-form#object-fields-ordering

Reply:
Oh, so this is more of a bug fix than a feature addition?